etna/experimental/classification/predictability.py

```diff
@@ -1,10 +1,14 @@
+import os
+import warnings
 from typing import Dict
 from typing import List
+from typing import Optional
 from urllib import request
 
 import numpy as np
 from sklearn.base import ClassifierMixin
 
+from etna.models.utils import get_known_hash, verify_file_hash
```

Comment:
It's a little bit strange that we pass information about experimental modul to stable one.